src/orion/core/worker/consumer.py

```diff
@@ -65,12 +69,12 @@ def consume(self, trial):
         :type trial: `orion.core.worker.trial.Trial`
 
         """
-        log.debug("### Create new directory at '%s':", self.dir)
-        temp_dir = self.experiment.working_dir is None
+        log.debug("### Create new directory at '%s':", self.working_dir)
+        temp_dir = self.working_dir is None
         prefix = self.experiment.name + "_"
-        suffix = "{}".format(trial.id) if self.dir else ""
+        suffix = "{}".format(trial.id) if self.working_dir else ""
```

Comment:
This will always be true. We can simply set it as `suffix = trial.id`, it's ok if the trial id is included in the tmp dir.